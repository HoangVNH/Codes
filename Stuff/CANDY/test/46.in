10
66
DVVXVVDXXDDXVVXDXVXDVXDXDDXVXXXVDVVVDDXDXXXDDDVXDXXDVDVDXXXDVVDDXX
93
VDDVVVDDXDVDVVDVXVVVDXVDXDDXVVDVXVXXDXDVXXVVXVDDVXDDVXVDVDDDXVXDDVXXDDVDXXDVXXVDVDVXVXXDXXXXD
81
DVXDDDDDVVXDVDDDXDXVVXVVXDDDDXVXDXVVXDXXVDDDVVXDXXXXXVVVDXVXDXVXVXVDXVDXXDDXVXVVD
92
VVDVDXXDVXXXVDDVVXVDDXVXXDXDDVDXXVXXVXXXVDXXDXVVDVXXVXDXDXDVDXVDVDDDVVXXVXVVDXVXVDVVVVXDVVVX
50
XDVXDXVDXDXVXVDVVXVDVDVVDVVDDXDDXVVDVDDXDDXXDDDXXD
85
DVDXDVXVVVDVXXDVDXXDXDVVVVXDVXXVVDDDXVDDDXDDXVDVXXXXDVDDVDDDDDVXVXXXXDVVXVVXXDVVXDVXD
64
DXXDVDDXDXXDDDDXXXVXVDVVXVVDXVVXXDVDDDDXXXVDXVVDDVXVDDXDDDDXDXXX
66
VDVXDVVDDDDXXDDDDVDDDXDXDDDDXVDVXXVXXDVDXVDVXXDVVVXDDDDXVDDDXVDXDD
91
VVXXVVVXXDVDXDXVVDDXXXDVVDDXXXVDDVDDXXXXDVVXXVXXXXVVDXDDDXXVVXVVXVVVXVXXVXXDVXXXDDDXVXDVXVX
52
DXVXDVDXVXDXVVXVVXVDVDVDVXVVVDVDDXDVDDDXDDDVVVXVXVVV
