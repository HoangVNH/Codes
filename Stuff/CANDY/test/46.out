65
92
80
91
49
84
63
65
90
51
